Integrations/EDL/EDL.py

```diff
@@ -76,25 +84,49 @@ def save_context(now: datetime, out_dict: dict):
     demisto.setIntegrationContext(out_dict)
 
 
-def find_indicators_to_limit(indicator_query: str, limit: int) -> list:
+def find_indicators_to_limit(indicator_query: str, limit: int,
+                             panos_compatible: bool = True, url_ps: bool = False) -> list:
     """
     Finds indicators using demisto.searchIndicators
     """
-    iocs, _ = find_indicators_to_limit_loop(indicator_query, limit)
+    iocs, _ = find_indicators_to_limit_loop(indicator_query, limit, panos_compatible=panos_compatible, url_ps=url_ps)
     return iocs[:limit]
 
 
-def find_indicators_to_limit_loop(indicator_query: str, limit: int, total_fetched: int = 0, next_page: int = 0,
-                                  last_found_len: int = PAGE_SIZE):
+def find_indicators_to_limit_loop(indicator_query: str, limit: int, total_fetched: int = 0,
+                                  next_page: int = 0, last_found_len: int = PAGE_SIZE,
+                                  panos_compatible: bool = True, url_ps: bool = False):
     """
     Finds indicators using while loop with demisto.searchIndicators, and returns result and last page
     """
     iocs: List[dict] = []
     if not last_found_len:
         last_found_len = total_fetched
     while last_found_len == PAGE_SIZE and limit and total_fetched < limit:
+        formatted_iocs = []
         fetched_iocs = demisto.searchIndicators(query=indicator_query, page=next_page, size=PAGE_SIZE).get('iocs')
-        iocs.extend(fetched_iocs)
+        if panos_compatible or url_ps:
+            for ioc in fetched_iocs:
+                ioc_value = ioc.get('value', '')
+                if url_ps:
+                    ioc_value = _PORT_RE.sub(r'\g<1>', ioc_value)
+                if panos_compatible:
+                    # protocol stripping
+                    ioc_value = _PROTOCOL_RE.sub('', ioc_value)
+                    # mix of text and wildcard in domain field handling
+                    ioc_value = _INVALID_TOKEN_RE.sub('*', ioc_value)
+                    # for PAN-OS *.domain.com does not match domain.com
+                    # we should provide both
+                    # this could generate more than num entries according to PAGE_SIZE
+                    if ioc_value.startswith('*'):
+                        ioc_object_copy = deepcopy(ioc)
+                        ioc_object_copy['value'] = ioc_value[2:]
```

Comment:
so we are sure the prefix is `*.` ? lets check for that in the condition so it will be more clear why we are slicing from index 2